src/CoreWCF.Primitives/src/CoreWCF/Channels/Message.cs

```diff
@@ -646,7 +646,7 @@ private static void ReadFromBodyContentsToEnd(XmlDictionaryReader reader, Envelo
             reader.MoveToContent();
         }
 
-        internal static bool ReadStartBody(XmlDictionaryReader reader, EnvelopeVersion envelopeVersion, out bool isFault, out bool isEmpty)
+        public static bool ReadStartBody(XmlDictionaryReader reader, EnvelopeVersion envelopeVersion, out bool isFault, out bool isEmpty)
         {
```

Comment:
As it's a static method, no state is needed. Duplicate code in calling class in WebHttp and leave internal

Reply:
This is addressed.